src/Components/test/E2ETest/ServerExecutionTests/InteropReliabilityTests.cs

```diff
@@ -0,0 +1,349 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Ignitor;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.E2ETesting;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETests.ServerExecutionTests
+{
+    public class InteropReliabilityTests : ServerTestBase<AspNetSiteServerFixture>
```

Comment:
I don't think `ServerTestBase` is the right base class here, because that in turn inherits `BrowserTestBase`, which encapsulates all the stuff around using Selenium. I see you had to take special steps to override `InitializeAsync` to make it skip Selenium-specific stuff. So overall it seems our layering doesn't match the use case.

It looks like you're not get anything out of using `ServerTestBase` anyway. You could more directly just receive a `ServerFixture` as a constructor param and then interact with the server that way. If you want, consider defining an `IgnitorTestBase` that does this.

Reply:
done